### PR TITLE
footer-scrollbutton-refactor

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
- Info about Issue or bug

Closes: #208, #202 , #193

#### Describe the changes you've made

I have added scroll to top button on every page, footer is also added and two folder in src/client have be combined and one 
folder named components is created.
## Screenshots
Screenshot (486)
Screenshot (487)

https://user-images.githubusercontent.com/68965499/115293529-1164db00-a175-11eb-9fa0-42baa7b305c1.mp4

### CODE REVIEW

File: client/src/components/homepage/homepage.js [hunk 1]
```diff
@@ -120,6 +121,7 @@ function LandingPageorange(props) {
                     </div>
                 </div>
             </div>
+            <div className="hfooter"><Footer /></div>
```
Comment (codewithvk) [inline client/src/components/homepage/homepage.js:124]: You should give proper class name ex:-
className='footer-components'

Comment (codewithvk) [reply client/src/components/homepage/homepage.js:124]: Change for all file className accordingly features/working.

File: client/src/components/SignUp/SignUp.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ const SignUp = ()=>{
         <div className="col-1">
             <label for="name">Username</label>
         </div>
-        <div class="col-2">
+        <div class="co-2">
```
Comment (codewithvk) [inline client/src/components/SignUp/SignUp.js:137]: if `col-2` is not working then please give the name `colm-2` etc

File: client/src/components/Footer/footer.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+            </div>
+            <div className='col-xl-4 col-lg-4 col-md-6 mb-30'>
+              <div className='footer-widget'>
+                <div className='footer-widget-heading'>
+                  <br />
+                  <h3>Useful Links</h3>
+                </div>
+                <ul>
+                  <li>
+                    <a href='/'>Home</a>
```
Comment (codewithvk) [inline client/src/components/Footer/footer.js:82]: Just Suggestion:- You can use Link from react-router.

**Review by codewithvk — CHANGES_REQUESTED** [review]
@ChitvanRamani22 Please do the above changes!

**Comment by codewithvk** [comment]
@ChitvanRamani22 Can you resolve conflict?

**Comment by ChitvanRamani22** [comment]
@codewithvk i have resolved the conflicts and also made the changes that you asked for.

**Pull request merged**